microshift_running_apps/microshift-deleting-resource-manifests.adoc

```diff
@@ -1,34 +1,39 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="microshift-deleting-resource-manifests"]
-= Deleting resource manifests
+= Deleting or upgrading Kustomize manifest resources
 include::_attributes/attributes-microshift.adoc[]
 :context: microshift-deleting-resource-manifests
 
 toc::[]
 
-{microshift-short} supports the deletion of resource manifests in the following situations:
+{microshift-short} supports the deletion of manifest resources in the following situations:
 
 * Manifest removal: Manifests can be removed when you need to completely remove a resource from the cluster.
 * Manifest upgrade: During an application upgrade, some resources might need to be removed while others are retained to preserve data.
 
-When creating new manifests, you can use resource manifest deletion to remove or update old objects, ensuring there are no conflicts or issues.
+When creating new manifests, you can use manifest resource deletion to remove or update old objects, ensuring there are no conflicts or issues.
 
 [IMPORTANT]
 ====
-Manifest files placed in the `delete` subdirectories are not automatically removed and require manual deletion.
+Manifest files placed in the `delete` subdirectories are not automatically removed and require manual deletion. Only the resources listed in the manifest files placed in the delete subdirectories are deleted.
 ====
 
 include::modules/microshift-manifests-deletion-overview.adoc[leveloffset=+1]
 
 [id="microshift-examples-of-usecase_{context}"]
-== Use cases for resource manifest deletion
+== Use cases for manifest resource deletion
 
-The following explains the use case in which the resource manifest deletion is used.
+The following explains the use case in which the manifest resource deletion is used.
```

Comment:
Suggestion - maybe add a noun here for clarity?
```suggestion
The following sections explain the use case in which the manifest resource deletion is used.
```

Reply:
Done. This is followed